src/main/java/org/candlepin/subscriptions/files/ProductUom.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2021 Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.files;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Objects;
+
+/**
+ * This object represents the combination of swatch product
+ * id and Measurement.Uom.  This should map to a unique metric
+ * ID.  The ProductUom / metricId is used in the product profile
+ * registry.
+ */
+@Getter
+@Setter
+public class ProductUom {
+
+    private String productId;
+    private String uom;
+
+    public ProductUom() {
+        // required for YAML
+    }
+
+    public ProductUom(String productId, String uom) {
+        this.productId = productId;
+        this.uom = uom;
+    }
+
+    @Override
```

Comment:
Should I remove this and hashCode, and add @EqualsAndHashCode and @ToString?

Reply:
Yes. In general, if there's nothing in those methods outside of the standard equals/hashcode pattern, then IMO it is safe to replace with the annotations.

Reply:
Okay, good.  Done.